MathLib/LinAlg/UnifiedMatrixSetters.cpp

```diff
@@ -0,0 +1,179 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include <cassert>
+#include "UnifiedMatrixSetters.h"
+
+#ifdef OGS_USE_EIGEN
+
+// Dense Eigen matrix/vector //////////////////////////////////////////
+
+namespace MathLib
+{
+
+void setMatrix(Eigen::MatrixXd& m,
+               Eigen::MatrixXd::Index const rows, Eigen::MatrixXd::Index const cols,
+               std::initializer_list<double> values)
```

Comment:
I also wondered at first. I think, I even tried it, but it did not work. But the STL has similar signatures, cf. http://en.cppreference.com/w/cpp/container/vector/vector.